showpass/Showpass-E2E-tests
# TA-22-Ensure that each notification type under 'Profile' can be turned on and off

**Description:**
End users should be able to turn on and off notifications.
This spec will be verifying that each notification type can be switched on and off by a toggle.
Steps:
- Navigate to https://beta.showpass.com/account/profile/update-payment/
- Click ‘Notifications’
- Ensure all notification types can be turned on and off by the corresponding toggles

**Jira ticket:**
https://showpass.atlassian.net/browse/TA-22

**How to test:**
Run the following test suite and ensure all tests pass
npx cypress run --spec "cypress/e2e/user-settings/profile.cy.js"

**Test execution results:**

<img width="1160" alt="Passed-4" src="https://github.com/showpass/Showpass-E2E-tests/assets/138615449/4e443629-4aa1-432b-8865-39d6e1a46f91">
